x/group/keeper/keeper.go

```diff
@@ -292,7 +292,6 @@ func (k Keeper) abortProposals(ctx context.Context, groupPolicyAddr sdk.AccAddre
 		return err
 	}
 
-	//nolint:gosec // "implicit memory aliasing in the for loop (because of the pointer on &proposalInfo)"
```

Comment:
seems like the linter doesnt want this to get remove

Reply:
we probably need to update the linter version